src/partials/nav-menu.hbs

```diff
@@ -2,7 +2,7 @@
 <div class="nav-panel-menu is-active" data-panel="menu">
   <nav class="nav-menu">
     {{#with @root.page.componentVersion}}
-    <h3 class="title"><a href="{{{relativize ./url}}}">{{./title}}</a></h3>
+    <h3 class="title">{{./title}}</h3>
```

Comment:
This removes a link in the navbar. I don't understand why it should be removed


![image](https://user-images.githubusercontent.com/27200110/214364296-e875d02f-5f01-469e-8f97-43142d68a1f6.png)
_previous navigation facilities_

Reply:
I made the (bad) assumption it was not useful.